bin/docker/alpine/Dockerfile

```diff
@@ -15,11 +15,10 @@ ADD . .
 RUN bin/build
 
 
-
 FROM alpine:3.6
 
 # Install packages
-RUN apk add --update --no-cache iptables ca-certificates openvpn bash sudo \
+RUN apk add --update --no-cache iptables ca-certificates openvpn bash sudo curl \
```

Comment:
I'm not sure that we need to add extra packages to the main image. If curl required somewhere else maybe create a new image for it?